### ISSUE TITLE
Request specs are needed

### ISSUE THREAD

**Issue by davidvandusen** [lighthouse-labs/compass-quiz#1]
As the software is rapidly prototyped, specs are needed for each request route so that changes to models, etc., don't break existing functionality.

Ideally these will be organized as request specs in a `spec/requests` directory, as described at https://github.com/rspec/rspec-rails#request-specs.

**Comment by varbark** [comment]
Work on it.

**Comment by varbark** [comment]
So the objective is that I should write each route's request spec? 
For example: 
:activities, only: [:index, :show, :create, :destroy, :update]
Then in my spec file will test each route for activities?

**Comment by davidvandusen** [comment]
Yeah, there should be at least one test for each of the routes. `rake routes` would be a good checklist to go from. The value of these tests will be that if a change is made that causes a page to fail to render, it will be apparent. Then future tests can be added to the spec files you create when bugs are encountered.
